pyproject.toml

```diff
@@ -0,0 +1,46 @@
+[tool.poetry]
+name = "tad"
+version = "0.1.0"
+description = ""
+authors = ["robbertuittenbroek <robbert.uittenbroek@rijksoverheid.nl>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.12"
+questionary = "^2.0.1"
+pyyaml = "^6.0.1"
+
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
+
+[tool.ruff]
+# Set the maximum line length to 127 characters.
+line-length = 120
+
+
+[tool.ruff.lint]
+select = ["I", "SIM", "B", "UP", "F", "E"]
+ignore = [] # List any rules to be ignored, currently empty.
+
+# Allow auto-fixing of all enabled rules when using the `--fix` option.
+fixable = ["ALL"]
+unfixable = [] # Specify rules that cannot be auto-fixed, if any.
+
+# Define a regex pattern for allowed unused variables (typically underscore-prefixed).
+dummy-variable-rgx = "^(_+|(_+[a-zA-Z0-9_]*[a-zA-Z0-9]+?))$"
+
+
+[tool.ruff.format]
+# Enforce double quotes for strings, following Black's style.
+quote-style = "double"
+
+# Use spaces for indentation, in line with Black's formatting style.
+indent-style = "space"
+
+# Keep magic trailing commas, a feature of Black's formatting.
+skip-magic-trailing-comma = false
+
+# Automatically detect and use the appropriate line ending style.
+line-ending = "auto"
```

Comment:
please at pyright

Reply:
Yeah, not as trivial as you'd hope. If you run pyright as pre-commit you'd need to make sure it takes the correct venv.

Reply:
I'd like to park this for later and find out how to do this right for all (future) projects